app/components/work_history_review_component.rb

```diff
@@ -1,4 +1,4 @@
-# Used in Candidate and Provider interface
+# Used in Candidate, Support and Provider interface
```

Comment:
I find it hard to trust these comments. The components not being namespaced is quite a strong signal they're available everywhere

Reply:
Agreed - removed!